XU-TechnicalPaper.md

```diff
@@ -137,7 +137,11 @@ A description of how we realize above goals:
 
 1. Minimize latency of order updates
 
-For orders, the DOB protocol strictly follows the first come, first served principle, which remains fully compatible with how order book systems of exchanges work. To get the best achievable latency between two nodes which intend to receive order updates from each other, we require an open connection between two nodes on Internet Protocol level. From a XU network topology point of view, we require a direct connection without intermediary hop with all nodes a node wants to receive order information from. Transport Layer design choices using socket connections and [efficient congestion control mechanisms](https://github.com/google/bbr) are currently in the works.
+For orders, the DOB protocol strictly follows the first come, first served principle, which remains fully compatible with how order book systems of centralized exchanges work. To get the best achievable latency between two nodes which intend to receive order updates from each other, XUD requires a direct socket connection between two nodes on Internet Protocol level without intermediary hop - a full mesh network. Direct payment channels are optional, payments can be routed through intermediary hops.
```

Comment:
In the second sentence, delete the second "between two nodes" phrase. Also I'm not sure if we should say our orderbook protocol "strictly" follows first come, first serve, as it's not something that could be enforced.

Reply:
Agree with both, fixed.